fdbclient/NativeAPI.actor.cpp

```diff
@@ -1038,7 +1038,7 @@ void setupNetwork(uint64_t transportId, bool useMetrics) {
 		networkOptions.logClientInfo = true;
 
 	g_network = newNet2(false, useMetrics || networkOptions.traceDirectory.present(), networkOptions.useObjectSerializer);
-	FlowTransport::createInstance(transportId);
+	FlowTransport::createInstance(true, transportId);
```

Comment:
I feel like this also means you should be fixing the createInstance API to make it so that your previous typo is a compilation error :|